kuka_kr270_support/urdf/kr270r2700_macro.xacro

```diff
@@ -0,0 +1,216 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+
+  <xacro:macro name="kuka_kr270r2700" params="prefix">
+    <xacro:include filename="$(find kuka_resources)/urdf/common_materials.xacro"/>
+    <!-- links -->
+    <link name="${prefix}base_link">
+      <inertial>
+        <origin xyz="-0.027804 0.00039112 0.14035" rpy="0 0 0"/>
+        <mass value="1572.9"/>
+        <inertia ixx="89.282" ixy="-0.47721" ixz="0.85562" iyy="107.51" iyz="0.0067576" izz="172.02"/>
+      </inertial>
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/visual/base_link.dae"/>
+        </geometry>
+        <xacro:material_kuka_pedestal />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/collision/base_link.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_1">
+      <inertial>
+        <origin xyz="-0.036811 -0.024697 0.56577" rpy="0 0 0"/>
+        <mass value="1385.5"/>
+        <inertia ixx="90.873" ixy="33.809" ixz="17.159" iyy="147.03" iyz="0.063634" izz="168.19"/>
+      </inertial>
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/visual/link_1.dae"/>
+        </geometry>
+        <xacro:material_kuka_pedestal />
```

Comment:
```suggestion
        <xacro:material_kuka_orange />
```

Reply:
Done in 492fd2c72c8306d27eb2032a7526703bf972684b